File: 백준/Bronze/2920. 음계/음계.java
import java.util.Scanner;

public class Main{
    
    public static void main(String []args){
       
        int count = 0;
       int music [] = new int [8];
       Scanner t = new Scanner(System.in);

       for(int i = 0; i < music.length; i++){
            music[i] = t.nextInt();
       }
        
       for(int k = 0; k < music.length; k++){
            if(music[k] == k + 1){
                count ++;
            }
            else if(music[k] == 8-k){
                count --;
            }      

       }

       if(count == 8){
        System.out.println("ascending");
       }
       else if(count == -8){
        System.out.println("descending");
       }
       else{
        System.out.println("mixed");
       }

       t.close();

    }
}

File: 백준/Bronze/2920. 음계/음계.cc
#include <stdio.h>
#define Size 8

int main(){

    int music[Size];
    int count = 0;

    for(int i = 0; i < Size; i++ ){
        scanf("%d",&music[i]);
    }

    for(int k = 0; k < Size; k++){
        if(music[k] == k+1 ){
            count++;
        }
    }
     for(int t = 0; t < Size; t++){
        if(music[t] == 8-t ){
            count--;
        }
    }

    if(count == 8){
        printf("ascending");
    }
    else if(count == -8){
        printf("descending");
    }else
        printf("mixed");

    return 0;

}